src/components/widgets/TryOutWidgetTabParameters.js

```diff
@@ -87,6 +87,10 @@ class TryOutWidgetTabParameters extends Component {
       value = this.props.headers.find(h => h.name === param.name).value
     }
 
+    if (value === undefined) {
+      value = param.default || ''
```

Comment:
Remove the `|| ''`  from the assignment. Not all the values are string, so this is not a good idea. Leave it undefined if no default is provided.

Reply:
Did it as suggested.